experiments/ClimaEarth/Project.toml

```diff
@@ -57,8 +57,8 @@ YAML = "ddb6d928-2868-570f-bddf-ab3f9cf99eb6"
 ArgParse = "1.1"
 ArtifactWrappers = "0.2"
 AtmosphericProfilesLibrary = "0.1"
-ClimaAnalysis = "0.5.3"
-ClimaAtmos = "0.24"
+ClimaAnalysis = "0.5"
+ClimaAtmos = "0.25"
```

Comment:
I think we're still compatible with v0.24, so we could keep this as `"0.24, 0.25"` (same for `perf/Project.toml`)